test/leveldown-test.js

```diff
@@ -1,5 +1,15 @@
 const test = require('tape')
 const testCommon = require('./common')
 const abstract = require('abstract-leveldown/test/leveldown-test')
+const leveldown = require('..')
 
 abstract.args(testCommon.factory, test)
+
+test('test database creation non-string location throws', function (t) {
+  t.throws(
+    leveldown.bind(null, {}),
+    /constructor requires a location string argument/,
+    'non-string location leveldown() throws'
+  )
+  t.end()
+})
```

Comment:
This build will fail since master has changed.

Reply:
Perhaps we should target commit refs?

Reply:
Funny thing, the tests doesn't fail.

Reply:
Just goes from

![screenshot from 2018-07-13 11-19-58](https://user-images.githubusercontent.com/308049/42683840-c303d910-868e-11e8-8ec1-8e6df81b340a.png)

to

![screenshot from 2018-07-13 11-19-21](https://user-images.githubusercontent.com/308049/42683868-d67ca2d8-868e-11e8-89b4-55bb4f3034b5.png)

Reply:
> Perhaps we should target commit refs?

I'll target a commit ref in next PR.